app/renderer/ui/components/main/sections/dataRequests/tabs/history/index.tsx

```diff
@@ -7,6 +7,7 @@ import { Services } from "app/renderer/services"
 import { CardDefault } from "app/renderer/ui/components/card"
 
 const svgImage = require("svg/dataRequestsHistory.svg")
+const layoutStyles = require("app/renderer/ui/components/main/sections/smartContracts/style.scss")
```

Comment:
I think here you may have wanted to import from `dataRequests`, not from `smartContracts`